src/main/java/org/opendevstack/provision/authentication/crowd/CrowdSecurityConfiguration.java

```diff
@@ -110,13 +116,22 @@ protected void configure(HttpSecurity http) throws Exception {
 
     if (isBasicAuthEnabled) {
       logger.info("Added Basic Auth entry point!");
-      sec.httpBasic().realmName(crowdApplicationName);
+      sec.httpBasic()
+          .realmName(crowdApplicationName)
+          .authenticationEntryPoint(new HttpStatusEntryPoint(HttpStatus.UNAUTHORIZED));
     }
 
     sec.addFilter(crowdSSOAuthenticationProcessingFilter())
         .authorizeRequests()
         .antMatchers(
-            "/", "/fragments/**", "/webjars/**", "/js/**", "/json/**", "/favicon.ico", "/login")
+            "/",
+            "/fragments/**",
+            "/webjars/**",
+            "/js/**",
+            "/json/**",
+            "/favicon.ico",
+            "/login",
+            "/nfe/**")
```

Comment:
what does nfe stand for? this abbreviation is often used but I don't know what it means.

Reply:
Ok got now that it stands for NewFrontEnd but I'd still write it out like this otherwise this information gets lost.

Reply:
NFE=new front end -> we are planning to rename this in a different change